core/domain/skill_services.py

```diff
@@ -232,156 +232,161 @@ def get_all_skill_summaries():
-            total_count: number. Total count of skill summaries.
     """
-    skill_summaries, next_cursor, more, total_count = (
-        get_skill_summaries_in_batches(
-            num_queries_to_fetch, cursor=cursor, sort_by=sort_by))
-    augmented_skill_summaries = [skill_domain.AugmentedSkillSummary(
-        summary.id,
-        summary.description,
-        summary.language_code,
-        summary.version,
-        summary.misconception_count,
-        summary.worked_examples_count,
-        summary.skill_model_created_on,
-        summary.skill_model_last_updated) for summary in skill_summaries]
+    skill_summaries, urlsafe_start_cursor, more = (
+        _get_skill_summaries_in_batches(
+            num_items_to_fetch, cursor, sort_by))
 
     assigned_skill_ids = (
         topic_services.
         get_all_skill_ids_assigned_to_some_topic_with_topic_details())
 
-    for augmented_skill_summary in augmented_skill_summaries:
-        skill_summary_id = augmented_skill_summary.id
-        if skill_summary_id in assigned_skill_ids:
-            augmented_skill_summary.update_topic_name(
-                assigned_skill_ids[skill_summary_id]['topic_name'])
-            augmented_skill_summary.update_classroom_name(
-                assigned_skill_ids[skill_summary_id]['classroom_name'])
-
-    return augmented_skill_summaries, next_cursor, more, total_count
-
-
-def filter_skills_by_status(augmented_skill_summaries, status):
+    augmented_skill_summaries = []
+    for skill_summary in skill_summaries:
+        topic_name = None
+        classroom_name = None
+        if skill_summary.id in assigned_skill_ids:
+            topic_name = assigned_skill_ids[skill_summary.id]['topic_name']
+            classroom_name = (
+                assigned_skill_ids[skill_summary.id]['classroom_name'])
+
+        augmented_skill_summary = skill_domain.AugmentedSkillSummary(
+            skill_summary.id,
+            skill_summary.description,
+            skill_summary.language_code,
+            skill_summary.version,
+            skill_summary.misconception_count,
+            skill_summary.worked_examples_count,
+            topic_name,
+            classroom_name,
+            skill_summary.skill_model_created_on,
+            skill_summary.skill_model_last_updated)
+        augmented_skill_summaries.append(augmented_skill_summary)
+
+    return augmented_skill_summaries, urlsafe_start_cursor, more
+
+
+def _filter_skills_by_status(augmented_skill_summaries, status):
     """Returns the skill summary dicts after filtering by status.
 
     Args:
         augmented_skill_summaries: list(AugmentedSkillSummary). The list
-        of augmented skill summaries.
+            of augmented skill summaries.
         status: str. The status of the skill.
 
     Returns:
         list(AugmentedSkillSummary). The list of AugmentedSkillSummaries
-        matching the given status.
+            matching the given status.
```

Comment:
Deindent by 4. (Note: this convention is only for Returns since each function returns only one thing.)

Reply:
Done.